config/checkstyle_non_main_files_suppressions.xml

```diff
@@ -22,4 +22,6 @@
     <!--Empty file cannot have any new line at the end. -->
     <suppress checks="NewlineAtEndOfFile" files="[\\/]test[\\/].*[\\/]checks[\\/]misc[\\/]newlineatendoffile[\\/]InputEmptyFile\.txt"/>
     <suppress checks="NewlineAtEndOfFile" files="[\\/]test[\\/].*[\\/]grammars[\\/]InputRegressionEmptyAst\.txt"/>
+    <!-- Input file with CR line separator -->
+    <suppress checks="NewlineAtEndOfFile" files="[\\/]test[\\/].*[\\/]astprinter[\\/]InputFullOfBlockCommentsCR\.java"/>
```

Comment:
As it is CR line separator it could not pass this check, but for testing purpose I need only CR, so we should leave this suppression.